src/main/java/org/broadinstitute/consent/http/db/mapper/LibraryCardMapper.java

```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.consent.http.db.mapper;
+
+import org.broadinstitute.consent.http.models.LibraryCard;
+import org.jdbi.v3.core.mapper.RowMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class LibraryCardMapper implements RowMapper<LibraryCard>, RowMapperHelper {
```

Comment:
This could be a lot simpler if we used `LinkedHashMapRowReducer` and used `@RegisterBeanMapper(value = LibraryCard.class)` in the DAO class. We wouldn't need to do any of the `hasColumn` checks and we could let jdbi figure out all of the field names with `rowView.getRow(LibraryCard.class)`